packages/sdks/javascript/tests/DevopnessApiClient.spec.ts

```diff
@@ -50,4 +55,20 @@ test("base URL is configurable on initialization", () => {
   expect(apiClient.users).toHaveProperty('passwords')
 
   expect(apiClient).toHaveProperty('variables')
+
+  expect(apiClient).toHaveProperty('onTokenExpired')
 })
+
+test("expired access_token should trigger callback function", async () => {
+  const mockRefreshTokenCallback = jest.fn(token => token);
+
+  const apiClient = new DevopnessApiClient();
+  apiClient.accessToken = 'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJPbmxpbmUgSldUIEJ1aWxkZXIiLCJpYXQiOjE3MTE3MjE4MTQsImV4cCI6MTcxMTYzNTQxNCwiYXVkIjoid3d3LmV4YW1wbGUuY29tIiwic3ViIjoianJvY2tldEBleGFtcGxlLmNvbSIsIkdpdmVuTmFtZSI6IkpvaG5ueSIsIlN1cm5hbWUiOiJSb2NrZXQiLCJFbWFpbCI6Impyb2NrZXRAZXhhbXBsZS5jb20iLCJSb2xlIjpbIk1hbmFnZXIiLCJQcm9qZWN0IEFkbWluaXN0cmF0b3IiXX0.Id1BkqYEZUEdPyu2BaTH9ulETZiZuSH9XJVi49up1Qw';
```

Comment:
Perhaps add a sample token that is closer to reality? :-) 

<img width="1070" alt="image" src="https://github.com/devopness/devopness/assets/1943318/81f208ca-d6c0-453d-968c-145853b585e0">


```suggestion
  apiClient.accessToken = 'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJhcGkuZGV2b3BuZXNzLmNvbSIsImlhdCI6MTcxMjYwOTI1NiwiZXhwIjoxNzQ0MTQ1MjU2LCJhdWQiOiJ3d3cubXktcHJvZHVjdC5jb20iLCJzdWIiOiJzb21lb25lQGV4YW1wbGUuY29tIiwic2NvcGUiOlsic2VydmVyOnJlYWQiLCJzZXJ2ZXI6Z2V0X3N0YXR1cyIsImFwcGxpY2F0aW9uOmRlcGxveSIsImFwcGxpY2F0aW9uOmVkaXQiLCJjcm9uam9iOnJlYWQiLCJjcm9uam9iOmRlcGxveSIsInNzbC1jZXJ0aWZpY2F0ZTpkZXBsb3kiXX0.pEVwmAznm75UFCAsTVEqsi40TjeZDUuN86Z6QdpMyaQ;
```

Reply:
Improved!